docs/adminguide/updating/index.rst

```diff
@@ -0,0 +1,49 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_adminguide_updating:
+
+.. cspell:words linenos literalinclude
+
+======================
+Parsec Server updating
+======================
+
+This guide covers the updating procedure for the `Parsec`_ server (also called `parsec-backend`).
+
+.. note::
+
+  We consider that you are using the same infrastructure configuration as in the :ref:`Parsec Server installation guide <doc_adminguide_hosting>`.
+  Adapt this guide to your own situation.
+
+.. _Parsec: https://parsec.cloud
+
+Updating to a newer version
+***************************
+
+.. warning::
+
+  Before updating to a newer version, we recommend that you backup the database,
+  In case you need to rollback the update.
+
+In this guide, we will consider that you are using the parsec-backend at the version ``2023-10-11-v2.16.0-rc.5.dev-b33d909`` and you want to update-it to ``v2.16.0``.
+
+1. Update the docker image's tag of the ``parsec-backend`` in docker-compose's file:
```

Comment:
```suggestion
1. Update the ``parsec-backend`` docker image tag in docker-compose file (`parsec-server.docker.yaml`):
```

Reply:
Done

PS: RST use double ` `` ` for code span